src/protocols/Pager/Pager.cpp

```diff
@@ -500,6 +525,15 @@ void PagerClient::write(uint32_t codeWord) {
   }
 }
 
+bool PagerClient::addressMatched(uint32_t addr) {
+  for (size_t i = 0; i < filterNumAddresses; i++) {
```

Comment:
This should probably check `filterAddresses != NULL` .

Reply:
Yep, will fix.

Reply:
Do we want some kind of error handling in this case, or do we just silently not match anything?

Reply:
It's a private method, so no need to to report any error and just always return false if addresses are NULL. However, we should make sure it cannot crash ;)